docs/decisions/0004-managing-django-settings.rst

```diff
@@ -0,0 +1,37 @@
+Status
+======
+
+Accepted
+
+
+Context
+=======
+
+Settings are confusing right now and there are way too many layers. We have
+multiple python settings file that "inherit" from each other. In some cases some
+layers of the inheritance tree also pull settings from config files leading to
+more confusion.
+
+
+Decision
+========
```

Comment:
Can you add a reference to the ADR/OEP for Remote config and explain how this ADR relates/adopts it?

Reply:
There was never an ADR or OEP written around remote config so I have nothing to reference.